packages/fluentui/react-component-ref/src/RefFindNode.tsx

```diff
@@ -71,7 +71,7 @@ export class RefFindNode extends React.Component<RefProps> {
   componentWillUnmount() {
     handleRef(this.props.innerRef, null);
 
-    delete this.prevNode;
+    this.prevNode = null;
```

Comment:
@miroslavstastny this part was a fix of memory leak. Is this change safe?

Reply:
Agree with @ecraig12345 - memory-leak wise, the two should be equal.